dbench/client.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+#
+# Environment variables used:
+#  - SERVER: hostname or IP-address of the NFS-server
+#  - EXPORT: NFS-export to test (should start with "/")
+
+echo "Client Script"
```

Comment:
nit: Was it intentional?

Reply:
Added this for testing purpose as this avoided scrolling log file to find start of client script. Should I remove this and send PR again?

Reply:
Thats okay.. was just curious :)